src/Shared/Microsoft.DotNet.Services.Utility/Microsoft.DotNet.Services.Utility.csproj

```diff
@@ -6,7 +6,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Options" />
+    <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="WindowsAzure.Storage" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.DncEng.CommandLineLib\Microsoft.DncEng.CommandLineLib.csproj" />
```

Comment:
The services utility project shouldn't reference the command line lib. That's a very strange dependency.

Reply:
Yeah, I was brining in SystemClock with it, I replaced it